lib/ui/components/Settings/SettingsSection.js

```diff
@@ -39,6 +39,16 @@ class SettingsSection extends React.Component {
             Always show Mimic buttons
           </label>
         </SettingsBlock>
+        <SettingsBlock>
+          <label htmlFor="disable-output">
```

Comment:
Please use either Div or Span components from lib/ui/components/common/base.js
To avoid css leakage

Reply:
Sorry, I'm not very experienced with React.

Do you mean to replace `<label htmlFor="disable-output">` with: `<Div>` or `<Div htmlFor="disable-output">`?

If yes, then the user won't be able to click the text to toggle the checkbox.

I used as an example the input above this one which also uses `<label>`(and we use `<label>` in more places in the ui in the same fashion).

Maybe we need to create a new `Label` component in `lib/ui/components/common/base.js`?